HUD/N_0xf83d0febe75e62c9.md

```diff
@@ -5,20 +5,26 @@ ns: HUD
 
 ```c
 // 0xF83D0FEBE75E62C9
-void _0xF83D0FEBE75E62C9(Any p0, Any p1, Any p2, Any p3, Any p4, Any p5, Any p6, Any p7);
+void _0xF83D0FEBE75E62C9(Blip blip, Any p1, Any p2, float width, Any p4, float length, float heading, Any p7);
```

Comment:
I almost forgot, when changing argument types, you should add `cs_type (Any)` annotations to whatever it is you're changing, it's for backward compatibility with old C# function parameter types (those that are changed).

Take a look at this example:
https://github.com/citizenfx/natives/blob/ba66873f1167eef2f4dfdcb7e4803d88264ccae7/PATHFIND/SetGpsDisabledZone.md?plain=1#L8

So all in all you would get:

```c
void _0xF83D0FEBE75E62C9(cs_type(Any) Blip blip, Any p1, Any p2, cs_type(Any) float width, Any p4, cs_type(Any) float length, cs_type(Any) float heading, Any p7);
```

Apologies for all the constant back and forth, thank you for your contribution.

Reply:
This is not needed for the hash native or does it? I'll add this nonetheless.

Reply:
> This is not needed for the hash native or does it? I'll add this nonetheless. Thanks for the solution!

Good question, only if return types are changed, but in this case the function type is still `void`, so no.
I'll approve this PR now, thanks! 😃